models/articles.model.js

```diff
@@ -0,0 +1,12 @@
+const db = require("../db/connection");
+
+function selectArticleById(id) {
+  return db.query(
+    `
+    SELECT * FROM articles
+    WHERE article_id = $1`,
+    [id]
+  );
+}
```

Comment:
You should ideally be accessing the result of the db query in a .then block here and then returning just the first item in the array